frontend/frontend/src/components/TheNavBar/components/AccountDropdownContents.vue

```diff
@@ -125,17 +126,20 @@
   <div v-else class="signed-in-dropdown">
     <b-dropdown-form>
       <b-form-group>
-        <b-form-text style="font-size: 1.1em; display: flex; flex-direction: column;">
+        <b-form-text
+          style="font-size: 1.1em; display: flex; flex-direction: column"
+        >
           <div class="first-row">
-            <div style="float: left; padding-top: 0.24em;">
-              User: 
-            </div>
+            <div style="padding-top: 0.24em; float: left">User:</div>
             <div class="username-text">
               {{ $store.state.activeUser.username }}
             </div>
           </div>
-          <div class="second-row" style="padding-top: 0.24em;">
-            Email: {{ $store.state.activeUser.email }}
+          <div class="second-row" style="padding-top: 0.24em">
+            <div style="float: left">Email:</div>
+            <div style="padding-left: 0.4em; float: left">
+              {{ $store.state.activeUser.email }}
```

Comment:
Sometimes these can act weirdly, worth making a computed: `email() { return this.$store.state.activeUser.email }`
```suggestion
              {{ email }}
```

Reply:
Is this for all situations where Vuex states/stores are used within the html?

Reply:
Are you sure about this bit? I do have:
 activeUser() {
      return this.$store.state.activeUser;
    },
as a computed property already

Reply:
Should I get rid of this and just use specific pieces of state data (eg. username) instead?